Makefile

```diff
@@ -30,9 +30,8 @@ include .bingo/Variables.mk
 ##@ Development
 
 .PHONY: generate
-generate: build # Generate CLI docs and samples
+generate: build # Generate CLI docs
```

Comment:
What CLI docs are we generating with this?

From what I can tell we don't have anything that I think would result from running this `go generate ./...` command but I could be wrong. Could you verify that we actually still need this make target? I have a feeling we might be able to get rid of it altogether

Reply:
I read `docs` and thought we might be doing something there, but I just searched for `go:generate` marker and it doesn't appear anything is in the code, so I'll remove this.